frontend/views/containers/chatroom/MessageActions.vue

```diff
@@ -121,7 +121,8 @@ export default ({
       }
     },
     type: String,
-    isMsgSender: Boolean
+    isMsgSender: Boolean,
+    isGroupCreator: Boolean
```

Comment:
Instead of passing this in via a parameter, maybe a computed property can be used instead to check `getters.ourIdentityContractId === getters.currentGroupOwnerID`?

Reply:
I would like to use the `isGroupCreator` property instead of adding a new computed variable in the `MessgeActions` component. The reason is because we are using `isGroupCreator` property in the `ChatAttachmentPreview` component either, so we should also make a computed variable in `ChatAttachmentPreview` component if we go this way.

Additionally, using a computed variable instead of prop requires to add 4 getters(`isDirectMessage`, `currentGroupOwnerID`, `ourIdentityContractId`, `currentChatRoomId`) which seems to be more untidy.